db/system_keyspace.cc

```diff
@@ -2121,6 +2119,11 @@ std::vector<schema_ptr> system_keyspace::all_tables(const db::config& cfg) {
         r.insert(r.end(), {topology(), cdc_generations_v3(), topology_requests()});
     }
 
+    if (cfg.check_experimental(db::experimental_features_t::feature::AUTH_V2)) {
+        auto auth_tables = db::system_auth_keyspace::all_tables();
+        std::copy(auth_tables.begin(), auth_tables.end(), std::back_inserter(r));
+    }
```

Comment:
it's added in system_keyspace::all_tables so that it can be registered properly in all places as system table, e.g. protected from user writes etc.

duplication in schema_loader is only for the purpose of this tool, for some reason it searches for schema this way.

you can see that system_keyspace::all_tables is actually multi keyspace - e.g. system_schema is also added there.